doc/source/user_guide/groupby.rst

```diff
@@ -1435,7 +1439,7 @@ use the ``pd.Grouper`` to provide this local control.
 
    df
 
-Groupby a specific column with the desired frequency. This is like resampling.
+Groupby a specific column with the wanted frequency. This is like resampling.
```

Comment:
Similar to the above. In addition, "wanted frequency" sounds awkward to me, although I don't know if I can express why.

Reply:
I will leave it as it was.